### ISSUE TITLE
CSS property: "max-width: stretch" doesn't throw error even if it is unsupported

### ISSUE THREAD

**Issue by egidijusja** [ismay/stylelint-no-unsupported-browser-features#137]
Currently I have stylelintrc config:
```
{
  "plugins": [
    "stylelint-no-unsupported-browser-features"
  ],
  "rules": {
    "plugin/no-unsupported-browser-features": [true, {
      "browsers": [
        "Chrome >= 76",
        "Safari >= 12",
        "Edge >= 18",
        "Firefox >= 72"
      ],
      "ignorePartialSupport": false
    }]
  }
}
```
And in one of my css files I have:
```
max-width: stretch;
```

`stretch` value is experimental feature and it does have partial support for chrome and edge and no support for safari and firefox (https://caniuse.com/?search=stretch), although when I try to lint it (`npx stylelint "./src/**/*.css"`) it doesn't show any errors or warnings.

**Comment by ismay** [comment]
Hi @egidijusja, thanks for the issue. https://github.com/anandthakker/doiuse, the library that this plugin uses for detecting css support, is the lib responsible for the actual detecting of support. This plugin only wraps it.

If you want to fix this or discuss it then the doiuse repo is the place to do it :). Hope that helps! I'll close this since there isn't anything I can do about it from this lib.
